src/main/java/com/orbit/frc/subsystems/leds/Leds.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Package: com.orbit.frc.subsystems.leds
+ * Description: This package contains classes related to controlling LEDs on a robot.
+ */
+
+package com.orbit.frc.subsystems.leds;
+
+import java.util.HashMap;
+
+import edu.wpi.first.wpilibj.Spark;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/*
+ * Class: Leds
+ * Description: This class represents the subsystem responsible for controlling LEDs on a robot.
+ */
+public class Leds extends SubsystemBase {
+    private final LedConfig config; // Configuration object for LEDs
+
+    private Spark LEDController; // Spark object for controlling LEDs
```

Comment:
move this to the config so the user can pass their own spark if theyd like

Reply:
confused about this, how would i do that

Reply:
take a look at other subsystems that use non-motor hardware & their configs

Reply:
all sybsystems haev motors :skull: